Recon/src/Recon/SequentialTrigger.cxx

```diff
@@ -0,0 +1,68 @@
+#include "Recon/SequentialTrigger.h"
+#include "Recon/Event/TriggerResult.h"
+
+namespace recon {
+
+/**
+ *Instantiates the variables used in this processor. We take trigger_list_ and trigger_passNames_ to obtain trigger collections, a doOR_ and doAND_ option to check if one of or all events pass, and doVAL_ to output a validation collection
+ * */
+
+void SequentialTrigger::configure(framework::config::Parameters &ps) {
+  trigger_list_ = ps.getParameter<std::vector<std::string>>("trigger_list");
+  trigger_passNames_ = ps.getParameter<std::vector<std::string>>("trigger_passNames");
+  doAND_ = ps.getParameter<bool>("doAND");
+  doOR_ = ps.getParameter<bool>("doOR");
+  doVAL_ = ps.getParameter<bool>("doVAL");
+  //Returns an error if some bad combination of doOR and doAND is enabled.
+  if (doAND_ == doOR_){
+    ldmx_log(fatal) << "Either tried to do both or neither of  doAND and doOR. Exiting.";
+    throw std::invalid_argument("Either tried to do both or neither of doAND and doOR. Exiting.");
+    return;
```

Comment:
Just use the framework exceptions here. This will end processing and print the exception in the same format as others.
```suggestion
    EXCEPTION_RAISE("InvalidArg","Either tried to do both or neither of doAND and doOR. Exactly one should be true.");
```